rvip_roi_parser/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(rvip_roi_parser)
+
+add_compile_options(-std=c++11)
+
+find_package(catkin REQUIRED COMPONENTS
+  darknet_ros_msgs
+  rospy
+  roscpp
+  std_msgs
+  sensor_msgs
```

Comment:
Sort.

Reply:
Fixed in commit [`
662c503 `](https://github.com/ros-industrial/rvip/pull/1/commits/662c50373d6b0965f440c81c7e822f428eb23e46).